README.md

```diff
@@ -376,6 +376,36 @@ To disable the splashscreen add the following preference to `config.xml`:
 <preference name="SplashScreenDelay" value="0"/>
 ```
 
+##### Display app version (Android, iOS)
+
+To display your app version on splashscreen add the following preference to `config.xml`:
+```xml
+<preference name="ShowSplashScreenAppVersion" value="true" />
+```
+
+Now just configure your app version into the widget version attribute, like that : 
```

Comment:
Is there a way to _not_ do that? I always thought the version was required - which would make calling this out a bit unnecessary. You could of course change the text in a way that just explains where it will be taking this value from.
(Also: superfluous space before `:` at the end of the line)

Reply:
I'm not sure to understand this part : `Is there a way to not do that?`

The widget `version` attribute is required (cf. https://cordova.apache.org/docs/en/latest/config_ref/index.html).

So, if i remove preference `ShowSplashScreenAppVersion`, the app version will be always displayed on splashscreen.
Some users don't want this behaviour.

What would i mean in the doc is that you need to add this preference to enable display app version.

If i'm wrong, do not hesitate to be more explicit ;)
Sorry, i need to improve my english.

Reply:
Sorry, I was not very clear:

Reply:
okay, thanks